openedx/core/lib/courses.py

```diff
@@ -44,3 +44,26 @@ def create_course_image_thumbnail(course, dimensions):
     _content, thumb_loc = contentstore().generate_thumbnail(course_image, dimensions=dimensions)
 
     return StaticContent.serialize_asset_key_with_slash(thumb_loc)
+
+
+def additional_image_url(course, image_key):
```

Comment:
Can we make these modifications to the existing `course_image_url` function above and use it instead?